blocks/api/parser.js

```diff
@@ -148,7 +149,13 @@ export function getBlockAttributes( blockType, innerHTML, attributes ) {
 		return getBlockAttribute( attributeKey, attributeSchema, innerHTML, attributes );
 	} );
 
-	return blockAttributes;
+	return applyFilters(
+		'blocks.getBlockAttributes',
```

Comment:
Do we want to document this or keep it "secret" for now :)

Reply:
>Do we want to document this or keep it "secret" for now :)

I'm fine to document. Will add.